km_api/know_me/tests/filters/test_emergency_contact_filter_backend.py

```diff
@@ -0,0 +1,26 @@
+from know_me import filters, models
+
+
+def test_filter_list_owner(api_rf, emergency_contact_factory):
```

Comment:
The tests for the filter backends should follow the format of the other filter backend tests. If you want an idea of what the filter should look like, see the emergency item filter backend in #121.